### PR TITLE
Add CDT & MBRN ics20s to Stargaze

### PR DESCRIPTION
**Pull request by triccs**
Here are the denom_traces for confirmation

CDT: https://rest.cosmos.directory/stargaze/ibc/apps/transfer/v1/denom_traces/B0263C28B6F44651F4596413B41FDB749EA010BD1220816DAC0ABF9947C1E806

MBRN: https://rest.cosmos.directory/stargaze/ibc/apps/transfer/v1/denom_traces/E94BB144B818CB8061F43E202BEA1E9273B87D6326C8C6F4E6AE71C62FD37854

### CODE REVIEW

File: stargaze/assetlist.json [hunk 1]
```diff
@@ -218,6 +218,94 @@
...
+          "denom": "CDT",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/B0263C28B6F44651F4596413B41FDB749EA010BD1220816DAC0ABF9947C1E806",
+      "name": "CDT",
+      "display": "CDT",
+      "symbol": "CDT",
+      "logo_URIs": {
```
Comment (JeremyParish69) [inline stargaze/assetlist.json:239]: Please do not add logo_URIs--this is a legacy property.

File: stargaze/assetlist.json [hunk 2]
```diff
@@ -218,6 +218,94 @@
...
+          },
+          "chain": {
+            "channel_id": "channel-0",
+            "path": "transfer/channel-0/factory/osmo1s794h9rxggytja3a4pmwul53u98k06zy2qtrdvjnfuxruh7s8yjs6cyxgd/ucdt"
+          }
+        }
+      ],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/CDT.svg",
```
Comment (JeremyParish69) [inline stargaze/assetlist.json:258]: you have the property png but the extension svg, which doesn't match and causing validation failure

File: osmosis/assetlist.json [hunk 3]
```diff
@@ -17974,6 +17974,25 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/COCA.png"
         }
       ]
+    },
+    {
+      "description": "Memecoin for The International Brane Wave",
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:17979]: Why re-registering BRNZ?

Comment (triccs) [reply osmosis/assetlist.json]: wasn't trying to, must've been an old commit

**Comment by triccs** [comment]
moved to  #4719 after deleting old commits and making the changes

**Pull request closed without merging**